src/Main.java

```diff
@@ -1,5 +1,70 @@
-public class Main {
+import java.util.Arrays;
+import java.util.Random;
+
+public  class Main {
     public static void main(String[] args) {
         System.out.println("Hello, World!");
+        System.out.println(" Задача№1 ");
+        int[] spending = generateRandomArray();
+        int sum = 0;
+        for (int expense : spending) {
+            sum += expense;
+        }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей.");
+        {
+
+
+            System.out.println(" Задача№2 ");
+            int[] expence = {36_000, 45_500, 39_985, 21_456, 21_477};
+            Arrays.sort(expence);
+            int max = expence[0];
+            int min = expence[0];
+            for (int i = 1; i < expence.length; i++){
+                if (expence[i] < min) {
+                    min = expence[i];
+                }
+                if (expence[i] > max){
+                    max=expence[i];
+                }
+            }
+            System.out.println(" Минимальная сумма трат за неделю составила " + min + " рублей ");
+            System.out.println(" Максимальная сумма трат за неделю составила " + max + " рублей ");
+
+            System.out.println(" Задача№3 ");
+            int[] monthExpence = {34567, 123458, 65390, 568806, 235541};
+            int total = 0;
+            for (int week : monthExpence) {
+                total = total + week;
+            }
+            float weekAverage = (total / (monthExpence.length));
```

Comment:
деление целочисленное, т.е. 5 / 2 будет 2.0 у тебя, а не 2.5. И скобки лишние
```suggestion
            float weekAverage = total / monthExpence.length;
```